rpcs3/Emu/Cell/lv2/sys_rsxaudio.cpp

```diff
@@ -1671,9 +1672,11 @@ void rsxaudio_backend_thread::backend_init(const rsxaudio_state& ra_state, const
 		backend->SetErrorCallback(std::bind(&rsxaudio_backend_thread::error_callback, this));
 	}
 
+	// TODO: properly handle dowmnix
```

Comment:
rsxaudio only needs downmix channel count. This cellAudio format logic is not really applicable here.